app/models/user.rb

```diff
@@ -80,6 +36,7 @@ def self.house_manager
   has_many :events, through: :attendances
   has_many :excuses
   has_many :fines
+  has_one  :position
```

Comment:
Style/SingleSpaceBeforeFirstArg: Put one space between the method name and the first argument.